requirements.txt

```diff
@@ -7,4 +7,11 @@ Pillow==9.4.0
 pytorch_lightning==1.8.1
 torch==1.13.1
 tqdm==4.64.0
-transformers[sentencepiece]==4.26.0
\ No newline at end of file
+transformers[sentencepiece]==4.26.0
+librosa==0.9.2
+crepe==0.0.13
+pretty-midi==0.2.10
+empy==3.3.4
+catkin-pkg==0.5.2
+rospkg==1.5.0
+tensorflow==2.12.0
```

Comment:
Why do you add tensorflow?

Reply:
the python "crepe" package depends on TensorFlow, which works on onset note classification

Reply:
But if it is a dependency, you don't need to explicitly add it.

Reply:
Yes, we need.

[INFO] [1683209841.508541]: Loading crepe full-model...
Traceback (most recent call last):
  File "/home/wang/workspace/master_project_2023_audio/devel/lib/marimbabot_audio/onset_detection.py", line 15, in <module>
    exec(compile(fh.read(), python_script, 'exec'), context)
  File "/home/wang/workspace/master_project_2023_audio/src/marimbabot/marimbabot_audio/scripts/onset_detection.py", line 542, in <module>
    detector = OnsetDetection()
  File "/home/wang/workspace/master_project_2023_audio/src/marimbabot/marimbabot_audio/scripts/onset_detection.py", line 152, in __init__
    self._init_detection_config()
  File "/home/wang/workspace/master_project_2023_audio/src/marimbabot/marimbabot_audio/scripts/onset_detection.py", line 113, in _init_detection_config
    crepe.core.build_and_load_model(self.crepe_model)
  File "/home/wang/workspace/master_project_2023_audio/env/lib/python3.8/site-packages/crepe/core.py", line 44, in build_and_load_model
    from tensorflow.keras.layers import Input, Reshape, Conv2D, BatchNormalization
ModuleNotFoundError: No module named 'tensorflow'

Reply:
Somehow, it is not included in the requirement of the crepe, we need to explicitly add it.